src/3d/shaders/mesh/mesh.geom

```diff
@@ -0,0 +1,160 @@
+#version 330 core
+
+layout( triangles ) in;
+layout( triangle_strip, max_vertices = 3 ) out;
+
+uniform mat4 viewportMatrix;
+uniform bool flatTriangles;
+
+in MeshVertex {
+    vec3 wordPosition;
```

Comment:
shouldn't this be `world` instead of `word` ?

Reply:
of course...